pom.xml

```diff
@@ -32,7 +32,9 @@
   </developers>
 
   <properties>
-    <powermock.version>1.5.5</powermock.version>
+    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
Now, minimum version is `1.580.1`? It was before 1.532 - this look wrong to me. Doesn't it?

What is the reason?